services/crypto_update.proto

```diff
@@ -171,8 +171,13 @@ message CryptoUpdateTransactionBody {
         VirtualAddress add = 20;
 
         /**
-         * The 20-byte EVM address of the virtual address that is being removed.
-         */
-        bytes remove = 21;
+        * The 20-byte EVM address of the virtual address that is being disabled and therefore removed from active use on the account.
```

Comment:
The only difference between disable and remove is that the former retains ownership and the latter allows reuse. Add a comment to each field about the impacts of address reuse.

Reply:
Will do when we circle back